test/e2e/scrapeconfig_test.go

```diff
@@ -216,7 +216,7 @@ func testPromOperatorStartsWithoutScrapeConfigCRD(t *testing.T) {
 
 	restarts, err := framework.GetPodRestartCount(context.Background(), ns, pl.Items[0].GetName())
 	require.NoError(t, err)
-	require.Lenf(t, restarts, 1, "expected to get 1 container but got %d", len(restarts))
+	require.Emptyf(t, restarts, "expected to get 1 container but got %d", len(restarts))
```

Comment:
This test was failing on my machine with the message "expected to get 1 container but got 1". On line 213 one Pod is explicitly expected.  The GetPodRestartCount will therefore always return a map with at least one container, which should have 0 restarts. The `restarts` map has therefore a length of at least 1. 

Emptyf checks whether the map is empty, which it shouldn't. The error msg is even more cryptic, as it refers to the length of the map, which should be one according to the message. But here we're checking whether the map's nil / empty? 

I can't get my head around as to how this tests works :smile: I just wanted to explain, why I made this change and I why I didn't consider it unnecessary.